entries/c/costplusdrugs.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Cost Plus Drugs": {
+    "domain":"costplusdrugs.com",
+    "contact": {
+      "twitter": "costplusdrugs",
+      "form": "https://costplusdrugs.com/contact/support/"
+    },
+    "categories": [
+      "health"
+    ],
+    "regions": [
```

Comment:
Are you sure the regions are correct?
It looks to me like this is a U.S.-centric service.